sysdig/resource_sysdig_secure_policy.go

```diff
@@ -107,6 +107,11 @@ func resourceSysdigSecurePolicy() *schema.Resource {
 										Required:         true,
 										ValidateDiagFunc: validateDiagFunc(validation.IntAtLeast(0)),
 									},
+									"name": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
```

Comment:
AFAIK the value can be either Optional or Computed, but not both. If the user is able to specify it, it has to be Optional, if the value is filled by the provider itself (as an output parameter), then it needs to be Computed.

Reply:
Apparently this is not the case.   When both a property is declared Optional and Computed, the value is only computed when not provided.

https://github.com/hashicorp/terraform/issues/21278

Reply:
Didn't know that, thanks for pointing out